### ISSUE TITLE
64GB SD card and emonSDexpand

### ISSUE THREAD

**Issue by rapzak** [openenergymonitor/emonpi#76]
Hi,

Seems like when i use 64gb SD card with the 30oct18 image for raspberry it runs fine until i run: sudo emonSDexpand

Then it will not start up correctly seems Data partition is gone - maybe more

/Kasper

**Comment by glynhudson** [comment]
Try running `$ git pull` on the `~emonpi` directory to ensure you have the latest version before running emonSDexpand, see: https://github.com/emoncms/usefulscripts/pull/16

Also you may need to wait 30 min or more for the process to take place

Can you verify using a Linux PC if the Data partition has indeed gone?

**Comment by rapzak** [comment]
Hi,
I tried 3 times with no luck.
I did not try to get latest script - however the image is not to old..?

I had a local screen on - it did seems like it did not start the process at all after reboot.

Here is a pic of the drive:
https://photos.app.goo.gl/spoPLFRGJLZTEoxz7

I have now taken a 16gb drive and installed on that - expanded that and cloned that to the 64gb...
Then i have space for some time and hopefully can expand it later.
